ros/finalarm_description/CMakeLists.txt

```diff
@@ -2,12 +2,16 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(finalarm_description)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS xacro)
 
 catkin_package()
 
 find_package(roslaunch)
 
+# build urdf files into the share directory for backwards compatibility
+file(GLOB xacro_files urdf/*.urdf.xacro)
+xacro_add_files(${xacro_files} INSTALL DESTINATION urdf)
```

Comment:
We can build xacro files so we don't have to run xacro at runtime. This helps maintain compatibility with existing moveit config.